packages/modules/inventory-next/src/migrations/Migration20240719123015.ts

```diff
@@ -0,0 +1,73 @@
+import { Migration } from "@mikro-orm/migrations"
+
+export class Migration20240719123015 extends Migration {
+  async up(): Promise<void> {
```

Comment:
Q: when in our modules we manage the migrations, shouldn't we get the snapshot updated as well? such that if another person update the schema and run the command to generate the migration we don't end up with massive SQL to remove/update/create a lot things which was not present in the snap?

Reply:
yes, I believe there are many out of sync. 
we should also generate a single migration before releasing.